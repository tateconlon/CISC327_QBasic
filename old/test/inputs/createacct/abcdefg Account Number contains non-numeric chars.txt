login
agent
createacct
abcdefg
logout