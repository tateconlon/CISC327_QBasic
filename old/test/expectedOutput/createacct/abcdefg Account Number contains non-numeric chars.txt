login as agent successful
Account number abcdefg not valid
Transaction summary written and system logged out successfully